doc/migrate-to-pymongo4.rst

```diff
@@ -974,12 +974,19 @@ subdocument containing a ``$ref`` field would be decoded as a
 Encoding a UUID raises an error by default
 ..........................................
 
-The default uuid_representation for :class:`~bson.codec_options.CodecOptions`,
+The default ``uuid_representation`` for :class:`~bson.codec_options.CodecOptions`,
 :class:`~bson.json_util.JSONOptions`, and
 :class:`~pymongo.mongo_client.MongoClient` has been changed from
 :data:`bson.binary.UuidRepresentation.PYTHON_LEGACY` to
 :data:`bson.binary.UuidRepresentation.UNSPECIFIED`. Attempting to encode a
 :class:`uuid.UUID` instance to BSON or JSON now produces an error by default.
+If you were using UUIDs previously, you will need to set your ``uuid_representation`` to
+:data:`bson.binary.UuidRepresentation.PYTHON_LEGACY` to avoid data corruption. If you do not have UUIDs,
+then you should set :data:`bson.binary.UuidRepresentation.STANDARD`. If you do not explicitly set a value,
+you will receive an error like this when attempting to encode a :class:`uuid.UUID`::
+
+    ValueError: cannot encode native uuid.UUID with UuidRepresentation.UNSPECIFIED. UUIDs can be manually converted...
+
```

Comment:
We should reiterate that uuid representation can be set on a client here, like this:
```
client = MongoClient(uri, uuidRepresentation="pythonLegacy")
client.test.test.insert_one({'uuid': uuid.uuid4()})
```